### ISSUE TITLE
Emails in form submissions do not have link to assets anymore

### ISSUE THREAD

**Issue by felixstoessel** [statamic/cms#6652]
### Bug description

Hello,

we use forms in Statamic and in some earlier versions of Statamic, we got an email with the form in the following layout:

**Name:** Foobar
**Email:** foo@bar.com
**Attachment:** https://bar.com/assets/attached_image_123.jpg

Now we got the same email, that get's send to us, but the attachment field does not show any permalink to the file:

**Name:** Foobar
**Email:** foo@bar.com
**Attachment:**

Important: In the backend under forms, I can click on the attachment field and see its content. I am unsure which template get's used for sending. I assume `automagic-email.antlers.html`.

### How to reproduce

My `resources/forms`:

```
title: Contact
honeypot: real
email:
  -
    to: foo@bar.com
    subject: 'Foobar'
```

My `resources/blueprints/forms`:

```
sections:
  main:
    display: Main
    fields:
      -
        handle: name
        field:
          input_type: text
          antlers: false
          display: Name
          type: text
          icon: text
          listable: hidden
          validate:
            - required
      -
        handle: email
        field: global.email
        config:
          validate:
            - required
            - email
      -
        handle: application
        field:
          mode: grid
          container: form-attachments
          restrict: true
          allow_uploads: true
          max_files: 3
          display: Attachment
          type: assets
          icon: assets
          listable: hidden
          width: 50
          validate:
            - required
```

### Environment

```yaml
Statamic 3.3.20 Pro
Laravel 8.83.19
PHP 7.4.30
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

runtime (new)

**Comment by jacksleight** [comment]
This should be fixed in [3.2.25](https://statamic.com/release-notes#3.3.25)

**Comment by duncanmcclean** [comment]
Closing as this issue should be fixed if you upgrade (well noticed @jacksleight). If you're still having issue after updating, comment on this issue and it can be re-opened.
